Marlin/src/lcd/ultralcd.cpp

```diff
@@ -3998,6 +4010,186 @@ void kill_screen(const char* lcd_msg) {
 
   /**
    *
+   * LED Menu
+   *
+   */
+
+  #if ENABLED(LED_CONTROL_MENU)
+
+    bool led_save_color =
+    #if ENABLED(LED_USER_PRESET_STARTUP)
+      true
+    #else
+      false
+    #endif
+    ;
+
+    uint8_t led_old_red, led_red_intensity = LED_USER_PRESET_RED,
+            led_old_green, led_green_intensity = LED_USER_PRESET_GREEN,
+            led_old_blue, led_blue_intensity = LED_USER_PRESET_BLUE
+            #if ENABLED(RGBW_LED) || ENABLED(NEOPIXEL_LED)
```

Comment:
Looks like the `led_old_...` globals can go away when debugging is removed.

Feel free to remove debugging any time. An LED control menu isn't rocket surgery. 😝

Reply:
Yes... but those are for saving a new "default" color by the user.

Reply:
The debug code has been removed.